docs/gpa.md

```diff
@@ -109,7 +109,9 @@ The addition of `period` to the `Milestone` building block is [under discussion]
 
 * Use 'open', 'selective' or 'limited' for *the procurement method that will be used* in `tender/procurementMethod`
 * If *it will involve negotiation*, add "negotiated" to `tender/procurementMethodDetails`
-* If *it will involve electronic auction* set `tender/techniques/hasElectronicAuction` to `true`
+* If *it will involve electronic auction*
+  * set `tender/submissionMethod` to `electronicAuction`
+  * set `tender/techniques/hasElectronicAuction` to `true`
```

Comment:
We can restore the old guidance. The 'electronicAuction' code is intended to be deprecated, because it isn't a submission method: https://github.com/open-contracting/standard/issues/322#issuecomment-325137100

Reply:
Done